client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -6,11 +6,16 @@
         :title="countersTitle"
         :data="countersData"
       />
-      <div slot="right">
-        <button type="button" disabled class="btn btn-secondary btn-settings">Settings</button>
-        <button type="button" class="btn btn-secondary py-0 btn-settings" @click="$emit('expand')">
-          <font-awesome-icon :icon="['fas', (expanded ? 'compress-alt' : 'expand-alt')]" />
-        </button>
+      <div class="d-flex align-items-center" slot="right">
+        <small class="mr-3 run-counter" v-if="getVariablesRunsCount">
```

Comment:
because you created a classname for it `.run-counter` why not add all those specification in the CSS

```HTML
<span class="run-counter" v-if="getVariablesRunsCount">
```
```CSS
.run-counter {
  color: $text-color-light;
  font-size: .9em;
  margin-right: 1em; /* Whatever mr-3 is equal to */
}

Reply:
My thougt on this will need a couple of hours and beers.
No worries about it, that's why I approved the PR. Maybe next year we will have time to agree on front-end architecture and styling.

Reply:
Yes, we had a conversation about this long time ago and since @mj3cheun was the only FE engineer at that time, we went with this styling. I think I personally prefer to have things separate and declared at the bottom of the page for each component to keep the template as clean as possible